src/audioPlayer.js

```diff
@@ -8,11 +8,17 @@ AUDIO_TOGGLES.forEach((audioToggle) => {
     audioToggle.addEventListener('click', setPlayer.bind(audioToggle))
 })
 
+const VIDEO_TOGGLES = document.querySelectorAll('[data-element=video-toggle]')
```

Comment:
Sorry, forgot to ask last time, on line 3:
`PLAYER.querySelector('source').src = '0'`
Why `0` instead of an empty string?

Reply:
Good question, I'm thinking we don't need that line at all actually